openlibrary/templates/type/list/view.html

```diff
@@ -217,30 +211,26 @@ <h3 class="booktitle">
                             - $commify(seed.work_count) work$("s" if seed.work_count != 1 else "")
                         </span>
 
-                        $if 'daisy_url' in ebook:
-                            <div class="type small"><a href="$ebook['daisy_url']" title="Protected DAISY">Download DAISY eBook for print disabled</a></div>
-
                         $if seed.last_update:
                             <span class="time">
                                 Last updated
                                 <span>$datestr(seed.last_update)</span>
                             </span>
+
+                        $if ctx.user and ctx.user.key == list.get_owner().key:
```

Comment:
Moved to left side, under content, so it is less likely to be accidentally clicked when a patron actually wants to interact with a CTA button.